demos/py_scripts/swash.py

```diff
@@ -33,14 +33,13 @@ def main(_):
 
     swash_sim = inductiva.fluids.SWASH()
 
-    output_path = swash_sim.run(
+    task = swash_sim.run(
         input_dir=FLAGS.sim_dir,
         sim_config_filename=FLAGS.sim_config_filename,
-        output_dir=FLAGS.output_dir,
```

Comment:
In these demos we should download the output with `task.get_output(FLAGS.output_dir)`.

Reply:
And keep the previous logs.